packages/babel-traverse/src/index.js

```diff
@@ -33,6 +33,7 @@ traverse.explode = visitors.explode;
 traverse.NodePath = require("./path");
 traverse.Scope    = require("./scope");
 traverse.Hub      = require("./hub");
+traverse.cache    = require("./cache");
```

Comment:
I think the core-js polyfill will serve us fine in this case since it does essentially what we were already doing, but I would love to avoid exposing this publicly since don't _think_ we need to?